### PR TITLE
Prettify the terminal output

### PR DESCRIPTION
**Pull request by sibasish14**
Screenshot : 
screenshot from 2017-12-25 20-13-57
Closes https://github.com/palash25/CoWaPS/issues/7

### CODE REVIEW

**Comment by palash25** [comment]
(Powered by [GitMate.io](https://gitmate.io))

**Comment by palash25** [comment]
@sibasish14 looks good just make it PEP8 compliant and I will merge it.

File: scrape.py [hunk 1]
```diff
@@ -37,19 +38,27 @@ def fetch(url):
     soup = BeautifulSoup(r.content, 'html.parser')
     user_info = soup.find_all('div', attrs={'class': 'stat'})
 
+    t = Texttable()
```
Comment (palash25) [inline scrape.py]: change `t` to `table`

File: scrape.py [hunk 2]
```diff
@@ -37,19 +38,27 @@ def fetch(url):
...
+    info_list = []
```
Comment (palash25) [inline scrape.py]: `[]` already suggest that its a list. Let's call it `profile_stats` or something like that

**Comment by sibasish14** [comment]
@palash25 changes done

**Comment by sibasish14** [comment]
@palash25 pushed again with requested changes.

**Pull request merged**